### PR TITLE
Add smp-cg-relnotes-020623

### PR DESCRIPTION
**Pull request by kat-enos**
CG release notes for the SMP drop on or about 02/06/23

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by deepakbolangady — APPROVED** [review]
I steered clear of the release notes that were already documented in the past. That left me one release note to give minor feedback on. Approving with the assumption that you will take a look at that minor comment.

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 1]
```diff
@@ -14,6 +14,96 @@ For Harness SaaS release notes, see [Harness SaaS Release Notes](https://develop
...
+  **Note:** If the `harness.io/skip-versioning: "true"` annotation is in a manifest, Harness does not track release versions.  
+
+- `DelegateGroup` is not removed from the database or the UI after the associated delegate is deleted. (DEL-3913)
+   The code was changed to ensure that an inactive delegate is deleted from the UI after seven days.
+
+- Avoid calling `closeChannel` more than one time during watcher shutdown. Invoking the method two times can cause the `logsAvoid` method to unnecessarily wait during executor shutdown. (DEL-4924)
+ 
+- Fixed a shell delegate issue that occurred on restart. The delegate name was not visible. (DEL-5443)
+
+- Functionality was added to send an alert within 30 minutes of delegate shutdown. If delegate groups are in use, the alert is sent only when all the delegate members of a group shut down. To use this functionality, enable the `INSTANT_DELEGATE_DOWN_ALERT` feature flag. (PL-30005, ZD-34766)
```
[the remark this answers is not in the document]
Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:98]:
```suggestion
- Functionality was added to send an alert within 30 minutes of delegate shutdown. If delegate groups are in use, the alert is sent only when all the delegate members shut down. To use this functionality, enable the `INSTANT_DELEGATE_DOWN_ALERT` feature flag. (PL-30005, ZD-34766)
```

[the remark this answers is not in the document]
Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:98]: The usage is fine, but I tightened it up a bit.

**Review by pratmit — APPROVED** [review]
LGTM with the rel notes coming from - https://harness.atlassian.net/wiki/spaces/PD/pages/21308309612/Release+Notes+Current+Gen-+02+03+2023

**Pull request merged**